.github/workflows/ci.yaml

```diff
@@ -30,32 +30,3 @@ jobs:
         run: make tests-flink
         env:
           FLINK_LIBS: ./flink_libs
-
-  integration-tests:
```

Comment:
I'd like to keep this as a separate CI check, to avoid clutter.

Reply:
are you sure? to me this is just another kind of test and can be part of `tests/`